.github/profile/README.md

```diff
@@ -0,0 +1,120 @@
+  </picture>
+</a>
+
+<!-- Made with [OSS Insight](https://ossinsight.io/) -->
+
+
+<!-- Copy-paste in your Readme.md file -->
+
+<a href="https://next.ossinsight.io/widgets/official/compose-org-overview-stats?activity=pull-requests&owner_id=189788385&period=past_12_months" target="_blank" style="display: block" align="left">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="https://next.ossinsight.io/widgets/official/compose-org-overview-stats/thumbnail.png?activity=pull-requests&owner_id=189788385&period=past_12_months&image_size=2x2&color_scheme=dark" width="177" height="auto">
+    <img alt="[Overview] pull-requests of eclipse-tractusx" src="https://next.ossinsight.io/widgets/official/compose-org-overview-stats/thumbnail.png?activity=pull-requests&owner_id=189788385&period=past_12_months&image_size=2x2&color_scheme=light" width="177" height="auto">
+  </picture>
+</a>
+
+<!-- Made with [OSS Insight](https://ossinsight.io/) -->
+
+<br>
+
+<!-- Copy-paste in your Readme.md file -->
+
+<a href="https://next.ossinsight.io/widgets/official/compose-org-active-contributors?activity=new&owner_id=189788385&period=past_12_months" target="_blank" style="display: block" align="right">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="https://next.ossinsight.io/widgets/official/compose-org-active-contributors/thumbnail.png?activity=new&owner_id=189788385&period=past_12_months&image_size=2x3&color_scheme=dark" width="273" height="auto">
+    <img alt="New participants of eclipse-tractusx - past 12 months" src="https://next.ossinsight.io/widgets/official/compose-org-active-contributors/thumbnail.png?activity=new&owner_id=189788385&period=past_12_months&image_size=2x3&color_scheme=light" width="273" height="auto">
+  </picture>
+</a>
+
+<!-- Made with [OSS Insight](https://ossinsight.io/) -->
+
+<!-- Copy-paste in your Readme.md file -->
+
+<a href="https://next.ossinsight.io/widgets/official/compose-org-overview-stats?activity=issues&owner_id=189788385&period=past_12_months" target="_blank" style="display: block" align="center">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="https://next.ossinsight.io/widgets/official/compose-org-overview-stats/thumbnail.png?activity=issues&owner_id=189788385&period=past_12_months&image_size=2x2&color_scheme=dark" width="177" height="auto">
+    <img alt="[Overview] issues of eclipse-tractusx" src="https://next.ossinsight.io/widgets/official/compose-org-overview-stats/thumbnail.png?activity=issues&owner_id=189788385&period=past_12_months&image_size=2x2&color_scheme=light" width="177" height="auto">
+  </picture>
+</a>
+
+<!-- Made with [OSS Insight](https://ossinsight.io/) -->
+
+## License
+
+- [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/) 
```

Comment:
Already in the repo and slightly confusing (is it just for this content of for the whole content of the org?) I'd remove it.
```suggestion

```